lib/org/chromium/webidl/URLExtractor.js

```diff
@@ -0,0 +1,60 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+'use strict';
+
+foam.CLASS({
+  package: 'org.chromium.webidl',
+  name: 'URLExtractor',
+
+  requires: ['org.chromium.webidl.IDLFileContents'],
+
+  properties: [
+    {
+      name: 'exclude',
+      // documentation: 'Excludes all URLs matching this array of expressions.',
+      postSet: function(_, value) {
+        var exps = value.map(function(item) {
+          return item.source;
+        });
+        this.excludeRegexp = exps.join('|');
+      },
+    },
+    {
+      name: 'include',
+      // documentation: 'Only include URLs matching this array of expressions.',
+      postSet: function(_, value) {
+        var exps = value.map(function(item) {
+          return item.source;
+        });
+        this.includeRegexp = exps.join('|');
+      },
+    },
+    {
+      name: 'excludeRegexp',
```

Comment:
I have changed it to `excludeRegexp_` and `includeRegexp_` respectively. I believe that's the FOAM way of representing internal properties, but I could be wrong. @mdittmer please chime in if it is something else